src/components/Modals/UploadDocumentModal.jsx

```diff
@@ -1,34 +1,38 @@
 // React Imports
 import React, { useState, useContext } from 'react';
 // Material UI Imports
-import Box from '@mui/material/Box';
 import Button from '@mui/material/Button';
-import Checkbox from '@mui/material/Checkbox';
+import ClearIcon from '@mui/icons-material/Clear';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import FileUploadIcon from '@mui/icons-material/FileUpload';
 import FormControl from '@mui/material/FormControl';
 import FormControlLabel from '@mui/material/FormControlLabel';
+import FormHelperText from '@mui/material/FormHelperText';
+import SearchIcon from '@mui/icons-material/Search';
+import Switch from '@mui/material/Switch';
 import TextField from '@mui/material/TextField';
-import { DatePicker } from '@mui/x-date-pickers/DatePicker';
 import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
 import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
-import FormHelperText from '@mui/material/FormHelperText';
 // Utility Imports
 import { runNotification } from '../../utils';
 // Custom Hook Imports
 import { useStatusNotification } from '../../hooks';
 // Component Imports
-import DocumentSelection from './DocumentSelection';
-import FormSection from './FormSection';
+import DocumentSelection from '../Form/DocumentSelection';
```

Comment:
Think you could Import DocumentSelection and FormSection from "../Form". We just need to have them included inside `/Form/index.js` for exports.